microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md

```diff
@@ -1,24 +1,24 @@
 ---
-description: Get started with Remote Debugging Windows 10 devices
-title: Get started with remote debugging Windows 10 Devices
+description: Get started with Remote Debugging Windows 10 and later devices
+title: Get started with remote debugging Windows 10 and later Devices
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.date: 04/23/2021
 ms.topic: article
 ms.prod: microsoft-edge
-keywords: microsoft edge, web development, f12 tools, devtools, remote, debugging, windows 10, windows, device portal
+keywords: microsoft edge, web development, f12 tools, devtools, remote, debugging, windows 10, Windows 10 and later, Windows 11, windows, device portal
 ---
```

Comment:
See if you can establish the "10 and later" context at the top of an article, "this article applies to 10 and later", establish that that scope applies to the scope of the entire article, then shorten to "Windows" in body of article.